guides/Model Support.md

```
@@ -3,9 +3,9 @@
 
 ---
 
-# This guide as moved to [our docs](https://docs.getkiln.ai/docs/models-and-ai-providers). This page may be out of date.
+# This guide as moved to [our docs](https://docs.kiln.tech/docs/models-and-ai-providers). This page may be out of date.
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '.'

(MD026, no-trailing-punctuation)